herbrain/pregnancy/page_content.py

```diff
@@ -355,31 +356,6 @@ def ai_hormone_prediction(mesh_explorer, show_legend=True):
         ],
     )
 
-    acknowledgements_text = dbc.Row(
-        [
-            html.P(
-                [
-                    "Our AI was trained on data from the study: Pritschet, Taylor, Cossio, Santander, Grotzinger, Faskowitz, Handwerker, Layher, Chrastil, Jacobs. Neuroanatomical changes observed over the course of a human pregnancy. (2024)",
-                ],
-                style={"fontSize": S.text_fontsize, "fontFamily": S.text_fontfamily},
-            ),
-        ],
-    )
-
-    if show_legend:
-        substructure_legend_row = [
-            dbc.Row(
-                [
-                    html.Img(
-                        src=get_asset_url("substructure_legend.png"),
-                        style={"width": "100%", "height": "auto"},
-                    ),
-                ]
-            )
-        ]
-    else:
-        substructure_legend_row = []
```

Comment:
see above

Reply:
The legend is wrong (the colors do not correspond to the substructures mentioned). So we can remove it.